Restfull API/api/requirements.txt

```diff
@@ -0,0 +1,16 @@
+astroid==1.6.1
+click==6.7
+colorama==0.3.9
+Flask==0.12.2
+Flask-WTF==0.14.2
+isort==4.3.4
+itsdangerous==0.24
+Jinja2==2.10
+lazy-object-proxy==1.3.1
+MarkupSafe==1.0
+mccabe==0.6.1
+pylint==1.8.2
+six==1.11.0
+Werkzeug==0.14.1
+wrapt==1.10.11
+WTForms==2.1
```

Comment:
Why do you have WTFForms in your requirements.txt? Are you using this?

Reply:
At the moment no.I initially thought I'd use forms